lib/fb/page.rb

```diff
@@ -23,22 +23,34 @@ def initialize(options = {})
       @access_token = options[:access_token]
     end
 
+    # @return [Hash] a hash of metrics mapped to their values.
+    # @param [Array<String, Symbol>] :metrics the metrics to fetch.
+    # @option [Date] :until only sum seven days before this date.
+    def get_weekly_metrics(metrics, options = {})
+      since_date = options.fetch :until, Date.today - 1
+      params = {period: :week, since: since_date, until: since_date + 2}
+      metrics = get_insights Array(metrics), params
+      metrics.map {|m| [m['name'].to_sym, m['values'].last.fetch('value', 0)]}.to_h
+    end
+
     # @return [Integer] the number of views of the page.
     # @param [Hash] options the options
     # @option [Date] :until only count the views until this day.
     def view_count(options = {})
-      views = insights 'page_video_views', period: :day, since: '1652 days ago'
-      views.select!{|v| v['end_time'] < options[:until].strftime('%Y-%m-%dT%H:%M:%S+0000')} if options[:until]
-      views.sum{|x| x.fetch('value', 0)} || 0
+      views = get_daily_metrics 'page_views_total', period: :day, since: '1652 days ago'
```

Comment:
changed the metric from `page_video_views` to `page_views_total`. We had the wrong one previously.